core/src/test/java/org/mskcc/oncotree/utils/TumorTypesUtilTest.java

```diff
@@ -102,6 +109,29 @@ public void testFullOncoTreeCodeListByCode() throws Exception {
                     failureReport.append(makeMismatchMessage(oncoTreeCode, "name", returnedName, expectedName));
                     failureCount = failureCount + 1;
                 }
+                MainType expectedMainType = expectedTumorType.getMainType();
+                MainType returnedMainType = returnedTumorType.getMainType();
+                if (expectedMainType == null || returnedMainType == null) {
```

Comment:
If either is null, then a null pointer exception could thrown on line 131 when the .getName() method is called